src/components/header/header.module.css

```diff
@@ -1,4 +1,34 @@
 .header {
-  font-size: 34px;
-  text-align: center;
+  padding: 17px 0;
```

Comment:
That's my mistake, I apologize. I misunderstood the color schemes. I decided to design a dark layout, and by default, my browser uses a dark theme, so I don't see any issues locally or in the live version since I'm working with a dark theme. As evidence, you can open my homework in another browser, let's say Firefox, after selecting a dark theme in the settings. I won't push fixes for now so that it doesn't seem like I've fixed everything :)